main.cpp

```diff
@@ -45,6 +52,44 @@ int main() {
                 .build();
     });
 
+    server.register_handler<HttpMethod::POST>("/search/{id, ip_address}", [](const HttpRequestContext &context) {
+        std::string player_id = context.path_params.at{"id"};
+        std::string player_address = context.path_params.at{"ip_address"};
+        PlayerInfo player(search_id, client_address);
+
+        if(!search_manager.check_for_active_search(player)){
+            return HttpResponse::new_builder().status(204).build();
```

Comment:
Follow the convention when chaining method calls. Having more than 1 method per line makes the expression hard to read